### PR TITLE
Call me daddy

### PR DESCRIPTION
**Pull request by spookydonut**
## About The Pull Request
Adds linting for top level procs that send signals

### CODE REVIEW

**Comment by JJRcop** [comment]
Your PR title is disgusting

**Comment by ExcessiveUseOfCobblestone** [comment]
daddy

File: code/game/machinery/porta_turret/portable_turret_cover.dm [hunk 1]
```diff
@@ -82,6 +82,7 @@
 	parent_turret.attack_animal(user)
 
 /obj/machinery/porta_turret_cover/attack_hulk(mob/living/carbon/human/user)
+	SHOULD_CALL_PARENT(0)
```
Comment (ninjanomnom) [inline code/game/machinery/porta_turret/portable_turret_cover.dm]: If you're not willing to go through the whole proc chain to make them all call parent don't bother adding the linting yet.

**Review by ninjanomnom — CHANGES_REQUESTED** [review]
You should know that no one wants to review this kind of mass change, break it up

**Pull request closed without merging**